Hủaıbaı/assets/minecraft/lang/toa_tqg.json

```diff
@@ -67,6 +67,19 @@
   "selectServer.deleteButton": "Shỉy",
   "selectServer.refresh": "Nỉqguo sủe",
   "selectServer.hiddenAddress": "(Shủı)",
+  "selectWorld.gameMode": "Lủaqcha",
+  "selectWorld.gameMode.survival": "Pó mîefa",
+  "selectWorld.gameMode.creative": "Pó bâı",
+  "selectWorld.gameMode.spectator": "Pó kâqsi",
```

Comment:
**sî** makes a bit more sense to me here given that, idk, the game has audio as well as visuals